tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrAllowed.hlsl

```diff
@@ -0,0 +1,19 @@
+// RUN: %dxc -T lib_6_7 %s | FileCheck %s
```

Comment:
I'm not sure I can move any of these into the verifier test suite, since the tests are so dependent on the target profile / entry point arguments, and the point of the test will be entirely missed if I don't involve that information. The fact that the entry point / target profile can't be passed to a verifier test discourages the use of that suite in this case.